packages/babel-traverse/src/scope/index.ts

```diff
@@ -535,7 +542,7 @@ export default class Scope {
    */
 
   isStatic(node: t.Node): boolean {
-    if (isThisExpression(node) || isSuper(node)) {
+    if (isThisExpression(node) || isSuper(node) || isTopicReference(node)) {
```

Comment:
A topic reference should be treated as a constant IdentifierReference, therefore it is static and pure.